zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/Keyspace.scala

```diff
@@ -24,31 +24,34 @@ import com.twitter.util.Duration
 import java.net.InetSocketAddress
 
 object Keyspace {
-  def zookeeperServerSets(
-    keyspaceName: String = "Zipkin",
-    hosts: Seq[(String, Int)],
-    path: String,
-    timeout: Duration,
-    stats: StatsReceiver = NullStatsReceiver): KeyspaceBuilder = {
+  // this code is not used
```

Comment:
not a big fan of commenting out logic we can always pull back out of version control